supertokens_python/recipe/multitenancy/interfaces.py

```diff
@@ -97,6 +101,21 @@ def __init__(self, providers: List[ProviderConfig]):
         self.providers = providers
 
 
+class AssociateUserToTenantOkResult:
+    def __init__(self, was_already_associated: bool):
+        self.was_already_associated = was_already_associated
+
+
+class AssociateUserToTenantErrorResult:
+    def __init__(self, status: str):
+        self.status = status  # FIXME: Create seperate errors for each kind
```

Comment:
Please do this in this PR - I will not merge it otherwise. And again, please don't submit PRs with stuff like FIXME and TODOs..

Reply:
Done